src/libs/PolicyUtils.ts

```diff
@@ -194,6 +194,9 @@ function getTagList(policyTags: OnyxCollection<PolicyTags>, tagKey: string) {
     return policyTags?.[policyTagKey]?.tags ?? {};
 }
 
+// This is to remove unnecessary escaping backslash in tag name sent from backend for "Parent: Child" type of tags.
+const unescapeColon = (tag: string) => tag?.replace(/\\{1,2}:/g, ':');
```

Comment:
I think for consistency let's use a named function here, and maybe rename to `getCleanedTagName`.